weaviate/values.yaml

```diff
@@ -326,6 +326,57 @@ envSecrets:
   # create a Kubernetes secret with AUTHENTICATION_APIKEY_ALLOWED_KEYS key and its respective value
   # AUTHENTICATION_APIKEY_ALLOWED_KEYS: name-of-the-k8s-secret-containing-the-comma-separated-api-keys
 
+# Configure offload providers
+offload:
+  s3:
+    enabled: false
+    # If one is using AWS EKS and has already configured K8s Service Account
+    # that holds the AWS credentials one can pass a name of that service account
+    # here using this setting.
+    # NOTE: the root `serviceAccountName` config has priority over this one, and 
+    # if the root one is set this one will NOT overwrite it. This one is here for
+    # backwards compatibility.
+    serviceAccountName:
+
+    envconfig:
+      # Configure bucket where data should be saved, this setting is mandatory
+      BACKUP_S3_BUCKET: weaviate-offload
```

Comment:
These parameters should start with OFFLOAD_*
The ones, from Mohamed's feature definition are OFFLOAD_S3_ENDPOINT, OFFLOAD_S3_BUCKET, OFFLOAD_TIMEOUT and S3_ENDPOINT_URL